dependencies.gradle

```diff
@@ -28,12 +28,12 @@ ext {
     revEmbeddedRedis = '0.6'
     revEurekaClient = '2.0.2'
     revGroovy = '4.0.9'
-    revGrpc = '1.57.2'
+    revGrpc = '1.66.0'
     revGuava = '33.2.1-jre'
     revHamcrestAllMatchers = '1.8'
     revHealth = '1.1.4'
     revPostgres = '42.7.2'
-    revProtoBuf = '3.21.12'
+    revProtoBuf = '3.25.3'
```

Comment:
could you try 3.25.5? I know upgrading anything related to protobuf and grpc can be quite a PITA due to what they pull vs what other libraries relying on them pull, but if 3.25.5 works, we should use that - it is vulnerability-free at the moment

Reply:
let see :D

Reply:
let's see 😄

Reply:
it seems fine to me 👍